pkg/ipam/node.go

```diff
@@ -18,6 +18,9 @@ package ipam
 import (
 	"context"
 	"fmt"
+	"github.com/cilium/cilium/operator/watchers"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/cache"
```

Comment:
These imports need to move lower down to an appropriate block.

Reply:
Thanks for checking this. Re-ordering imports into 3 blocks (standard lib, project packages and external)